msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/wpj/AbstractWpjTest.java

```diff
@@ -92,19 +98,29 @@ public ITestBroker getBroker() {
     }
 
     /**
-     * Run LabUserQuery to get a cloud device administrator user
+     * Run LabUserQuery to get a cloud device administrator user.
      *
-     * @return cloud device administrator user
+     * @return cloud device administrator user.
      */
     public String getLabUserForCloudDeviceAdmin() {
-        final LabUserQuery query = new LabUserQuery();
-        query.userRole = LabConstants.UserRole.CLOUD_DEVICE_ADMINISTRATOR;
-        return LabUserHelper.loadUserForTest(query);
+        final LabApiAuthenticationClient authenticationClient =
+                new LabApiAuthenticationClient(BuildConfig.LAB_CLIENT_SECRET);
+        final LabClient labClient = new LabClient(authenticationClient);
```

Comment:
We can probably create these as `protected` at the class level

Reply:
0cd9fc24a68b9fb8e83bf85dad86af4e8f5c23b3